hack/tools/install.sh

```diff
@@ -153,9 +153,11 @@ function install_helm() {
 }
 
 function install_kops() {
+  # Build from source so we can test latest Kubernetes version earlier.
   INSTALL_PATH="${1}"
 
-  install_binary "${INSTALL_PATH}" "https://github.com/kubernetes/kops/releases/download/${KOPS_VERSION}/kops-${OS}-${ARCH}" "kops"
+  # Lower max processes to avoid oom-killed
+  GOMAXPROCS=1 install_go "${INSTALL_PATH}" "k8s.io/kops/cmd/kops@${KOPS_COMMIT}"
 }
```

Comment:
Would it be best to only install kOps using a commit if is explicitly defined, otherwise fall back to the existing logic?

```
install_binary "${INSTALL_PATH}" "https://github.com/kubernetes/kops/releases/download/${KOPS_VERSION}/kops-${OS}-${ARCH}" "kops"
```

Reply:
I would suggest against this change, as it would bloat the file and I'm not really sure what the usecase would be

Reply:
Seems relatively useful to be able to test against specific release tags, as we have been doing, unless there are extenuating circumstances for which we'd want to override and use a specific commit instead. Anyways, we could effectively accomplish the same by specifying the hash associated with the relevant tag - its only a minor inconvenience to cross check that, so I'm aligned with removing the previous code path.